docker/README.md

```diff
@@ -0,0 +1,254 @@
+
+### Base Builder: nuxeo/builder
+
+It provides:
+- A Nuxeo server distribution with appropriate permissions in the `/distrib` directory.
+- An `install-packages.sh` script to install Nuxeo packages.
+
+It is based on an OpenJDK image as it requires Java for the package installation.
+
+It must be used within a multi-stage build.
+For instance, you can use the following Dockerfile sample to build an image based on a `BASE_IMAGE`, containing a Nuxeo server distribution:
+- In the `NUXEO_HOME` directory.
+- With some Nuxeo packages installed, whose ZIP files must be located in the `local/packages` directory.
+- Owned by the UID user and GID group.
+
+```
+FROM nuxeo/builder:VERSION as builder
+COPY local/packages /packages
+RUN install-packages.sh /packages
+
+FROM BASE_IMAGE
+COPY --from=builder --chown=UID:GID /distrib NUXEO_HOME
+``` 
+
+### Platform Builder: nuxeo/builder-platform
+
+This image is similar to the [nuxeo/builder](#base-builder:-nuxeo/builder) image, in addition to which it has a set of Nuxeo packages pre-installed in the Nuxeo server distribution. These packages are described by the Maven project's [POM](builder-platform/pom.xml).
+
+In the same way, it must be used within a multi-stage build.
+For instance, you can use the following Dockerfile sample to build an image based on a `BASE_IMAGE`, containing a Nuxeo Platform distribution:
+ - In the NUXEO_HOME directory.
+ - With the set of pre-installed Nuxeo packages mentioned above.
+ - Owned by the UID user and GID group.
+
+```
+FROM nuxeo/builder-platform:VERSION as builder
+
+FROM BASE_IMAGE
+COPY --from=builder --chown=UID:GID /distrib NUXEO_HOME
+```
+
+### Nuxeo Base: nuxeo/base
+
+This image cas be used as the `BASE_IMAGE` in the Dockerfile samples seen above.
+
+Based on CentOS 7, it includes:
+- OpenJDK.
+- A `nuxeo` user with the `900` fixed UID.
+- The directories required to have the Nuxeo configuration, data and logs outside of the server directory, with appropriate permissions.
+- An entrypoint script to configure the server.
+- The default recommended volumes.
+- The environment variables required by the server, typically `NUXEO_HOME` and `NUXEO_CONF`.
+- The exposed port `8080`.
+
+It doesn't contain the Nuxeo server distribution itself.
+To build an image containing a Nuxeo server distribution with some packages installed, you must use a multi-stage build
+with the [nuxeo/builder](#base-builder:-nuxeo/builder) (or [nuxeo/builder-platform](#platform-builder:-nuxeo/builder-platform)) image and the [nuxeo/base](#nuxeo-base:-nuxeo/base) image, as in the following Dockerfile sample:
+
+```
+FROM nuxeo/builder:VERSION as builder
+COPY local/packages /packages
+RUN install-packages.sh /packages
+
+FROM nuxeo/base:VERSION
+RUN yum -y install ...
+COPY --from=builder --chown=900:0 /distrib $NUXEO_HOME
+USER 900
+```
+
+## Final Images
+
+### Slim: nuxeo/nuxeo
+
+It includes a bare Nuxeo server distribution without any package installed.
```

Comment:
maybe "any Nuxeo package installed" just to be clear between OS and Nuxeo packages

Reply:
Yes for the renaming.
Maybe indeed, a possible use case is running some server REST API tests.

Reply:
And this slim image is the simplest example of an image built using multi-stage from the `nuxeo/builder` and `nuxeo/base` images.